### ISSUE TITLE
Automatic language switching doesn't work for aria-label

### ISSUE THREAD

**Issue by SamKacer** [nvaccess/nvda#16285]
### Steps to reproduce:

```html

### Actual behavior:
<!--
Use "Speak command keys" (NVDA+4) and speech viewer to copy and paste here. 
Use braille viewer to copy and paste here.
You may additionally include an explanation.
-->

### Expected behavior:
<!--
Use "Speak command keys" (NVDA+4) and speech viewer to copy and paste here. 
Use braille viewer to copy and paste here.
You may additionally include an explanation.
-->

### NVDA logs, crash dumps and other attachments:

### System configuration
#### NVDA installed/portable/running from source:

#### NVDA version:

#### Windows version:

#### Name and version of other software in use when reproducing the issue:

#### Other information about your system:

### Other questions
#### Does the issue still occur after restarting your computer?

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

#### If NVDA add-ons are disabled, is your problem still occurring?

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

**Comment by SamKacer** [comment]
Apologies, submitted prematurely, I am editing the comment now

**Comment by SamKacer** [comment]
Ok, finished editing initial comment

**Comment by Adriani90** [comment]
Probably related to #15061.
